src/main/java/com/bfrisco/benchants/utils/Toggle.java

```diff
@@ -0,0 +1,77 @@
+package com.bfrisco.benchants.utils;
+
+import org.bukkit.*;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import java.util.List;
+
+public class Toggle implements Listener {
+
+
+
+    @EventHandler
+    public static void activateClick(PlayerInteractEvent event) {
+
+        Player player = event.getPlayer();
+        Material coolDown = Material.JIGSAW;
+        if (!event.getAction().isRightClick()) return; //should reduce the amount of times the interactevent continues to use resources
+        ItemStack item = player.getInventory().getItemInMainHand();
+        if (!ItemInfo.isTitanTool(item)) return;
+        if (!ItemInfo.isImbued(item)) return;
+        if (player.hasCooldown(coolDown)) return;
+        player.setCooldown(coolDown,25);
+        player.sendMessage("Made it past isImbued check in toggle event");
+
+        if (!player.isSneaking()) return; //should reduce the amount of times the interactevent continues to use resources
+
+        if (!player.hasPermission("benchants.toggle")) return;
+
+        toggleAncientPower(item,player);
+        new BEnchantEffects().toggleEffect(player);

```

Comment:
Do you think it would be better just to toggle the particle effect in the `toggleAncientPower` method?
I believe this is currently the only place where it is called, but in the future if it was ever called somewhere else we then would not have to duplicate the `toggleEffect` here.